common/persistence/visibility/store/elasticsearch/visibility_store.go

```diff
@@ -65,7 +65,6 @@ const (
 var defaultSorter = []elastic.Sorter{
 	elastic.NewFieldSort(searchattribute.CloseTime).Desc().Missing("_first"),
 	elastic.NewFieldSort(searchattribute.StartTime).Desc().Missing("_first"),
-	elastic.NewFieldSort(searchattribute.RunID).Desc().Missing("_first"),
```

Comment:
You need to change `setDefaultFieldSort` too.

Reply:
I just read PR description. I believe it should be removed from `setDefaultFieldSort` in this PR.

Reply:
Tried and I feel we should not do it. So I reverted the change. There are two reasons:
(1) We are disabling user specified orderby by default. This is not an issue if that's not turned on.
(2) If user intentionally turn it on, then removing the RunId as tie-breaker can cause a behavior change, if user does not specify all tie-breakers in order by (very likely the case, e.g. if they order by some custom search attributes with low cardinality).

Reply:
Default value for disable ORDER BY is `false`. But I agree that if custom order is specified then query is already slow. Adding `RunId` won't hurt much.